### DISCUSSION TITLE
Where can I find the privacy policy of Supabase?

### DISCUSSION THREAD

**Discussion by inbytesdevteam** [supabase/supabase#212] [Questions] [answered]
Do we have one? I'm planning to integrate this in our privacy policy and would like to reference Supabase privacy policy in case they want to learn more on how the data is stored. Thx

**Comment by kiwicopple** [comment]
Hey @inbytesdevteam - bear with us as we're going through it all with our lawyers. They don't ship as fast as we do. 

Internally, we have this as a required task for our move from Alpha to Beta, so it will be done at least before December.

**Comment by erkeen** [comment, marked as answer]
Hey @inbytesdevteam - You might be interested in the recent privacy policy of supabase. Please check this out https://supabase.io/docs/company/privacy

**Reply by inbytesdevteam** [reply]
Thank you @erkeen 😊
